README.md

```diff
@@ -1,42 +1,36 @@
-# Knative Client
+# Kn
 
-This section outlines best practices for the Knative developer experience. It is a reference for Knative CLI implementation, and a reference for Knative client libraries.
+The Knative client `kn` is your door to the [Knative](https://knative.dev) world.
```

Comment:
Why is this file changed in addition to the #677

Reply:
sorry, I mixed up to PRs. Should be fixed now.